draft-ietf-pquip-pqc-engineers.md

```diff
@@ -484,11 +466,9 @@ Falcon [Falcon] is based on the GPV hash-and-sign lattice-based signature framew
 
 The main design principle of Falcon is compactness, i.e. it was designed in a way that achieves minimal total memory bandwidth requirement (the sum of the signature size plus the public key size). This is possible due to the compactness of NTRU lattices.  Falcon also offers very efficient signing and verification procedures. The main potential downsides of Falcon refer to the non-triviality of its algorithms and the need for floating point arithmetic support in order to support Gaussian-distributed random number sampling where the other lattice schemes use the less efficient but easier to support uniformly-distributed random number sampling.
 
-Access to a robust floating-point stack in Falcon is essential for accurate, efficient, and secure execution of the mathematical computations involved in the scheme. It helps maintain precision, supports error correction techniques, and contributes to the overall reliability and performance of Falcon's cryptographic operations as well makes it more resistant to side-channel attacks.
-
-Falcon's signing operations require constant-time, 64-bit floating point operations to avoid catastrophic side channel vulnerabilities. Doing this correctly is very difficult and is also platform-dependent to an extreme degree, as NIST's report noted. Providing a masked implementation of Falcon also seems impossible, per the authors at the RWPQC 2023 symposium.
+Implementors of Falcon will need to be aware that Falcon signing is highly susceptible to side-channel attacks unless constant-time 64-bit floating point operations are used, which is extremely platform-dependent, as NIST's report noted.
```

Comment:
NEW: Implementers of Falcon need to be aware that Falcon signing is highly susceptible to side-channel attacks, unless constant-time 64-bit floating-point operations are used. This requirement is extremely platform-dependent, as noted in NIST's report.

Reply:
Done.